tools/accuracy_checker/accuracy_checker/annotation_converters/mrlEyes_2018_01.py

```diff
@@ -0,0 +1,85 @@
+"""
+Copyright (c) 2019 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import re
+
+from ..config import PathField
+from ..representation import ClassificationAnnotation
+from ..utils import get_path, read_txt
+
+from .format_converter import BaseFormatConverter, ConverterReturn
+
+
+class SampleConverter(BaseFormatConverter):
```

Comment:
please give more descritive name for converter class

Reply:
Done